cynic-querygen/src/query_parsing/normalisation.rs

```diff
@@ -571,6 +575,55 @@ mod tests {
         )
     }
 
+    #[test]
+    fn check_field_selected() {
+        let schema = load_schema();
+        let type_index = Rc::new(TypeIndex::from_schema(&schema));
+        let query = graphql_parser::parse_query::<&str>(
+            r#"
+           query MyQuery {
+              allFilms(after: "") {
+                edges {
+                  cursor
+                  node {
+                    created
+                    edited
+                    episodeID
+                  }
+                }
+              }
+            }
+            "#,
+        )
+        .unwrap();
+        let normalised = normalise(&query, &type_index).unwrap();
+
+        insta::assert_debug_snapshot!(normalised);
```

Comment:
I'm not sure if we even need the full snapshot on this one - think we just want to make sure that `noramlise` works, so could probably make this

```suggestion
        assert_matches!(normalise(&query, &type_index), Ok(_));
```

That way we don't end up with a 3000 line snapshot file, but still test the thing we care about.